boards/nxp/imx93_evk/imx93_evk_mimx9352_a55.dts

```diff
@@ -145,3 +154,14 @@
 &gpio4{
 	status = "okay";
 };
+
+&flexcan2 {
+	pinctrl-0 = <&flexcan2_default>;
+	pinctrl-names = "default";
+	sample-point = <875>;
+	sample-point-data = <875>;
```

Comment:
Please remove these. This will cause the CAN timing calculations to use the default values.